tests/test_default_qubit.py

```diff
@@ -673,6 +683,25 @@ def reference(*x):
             elif g == 'Hermitian':
                 self.assertAllEqual(circuit(H), reference(H))
 
+    def test_two_qubit_observable(self):
+        """Tests expval for two-qubit observables """
+        self.logTestName()
+        dev = qml.device('default.qubit', wires=2)
+
+        @qml.qnode(dev)
+        def circuit(x, target_observable=None):
+            qml.RX(x[0], wires=0)
+            qml.RY(x[1], wires=0)
+            qml.RZ(x[2], wires=0)
+            qml.CNOT(wires=[0, 1])
+            return qml.expval.Hermitian(target_observable, wires=[0, 1])
+
+        target_state = 1/np.sqrt(2) * np.array([1,0,0,1])
+        target_herm_op = np.outer(target_state.conj(), target_state)
+        weights = np.array([0.5, 0.1, 0.2])
+        expval = circuit(weights, target_observable=target_herm_op)
+        assert(np.allclose(expval,0.590556))
```

Comment:
Since this test is still using unittest, we'll need to use the standard unittest-provided assertions methods:
```suggestion
        self.assertAlmostEqual(expval, 0.590556, delta=self.tol)
```

Reply:
Side note: since the test suite is executed by pytest, it is fine to add new pytest-style tests to an existing unittest module! This might save time when porting later. For example, see https://github.com/XanaduAI/pennylane/blob/qgt/tests/test_qnode.py